R/secret.R

```diff
@@ -24,11 +26,13 @@
 #'     as an env var (e.g. `{MYPACKAGE}_KEY`) by adding a line to your
 #'     `.Renviron`.
 #'
-#' 2.  Encrypt strings with `secret_encrypt()` and other data with
-#'     `secret_write_rds()`, setting `key = "{MYPACKAGE}_KEY"`.
+#' 2.  Encrypt strings with `secret_encrypt()`, files with
+#'     `secret_encrtype_file()`,  and other data with `secret_write_rds()`,
+#'     setting `key = "{MYPACKAGE}_KEY"`.
 #'
-#' 3.  In your tests, decrypt the data with `secret_decrypt()` or
-#'     `secret_read_rds()` to match how you encrypt it.
+#' 3.  In your tests, decrypt the data with `secret_decrypt()`,
+#'     `secret_decrypt_file()`, or `secret_read_rds()` to match how you encrypt
+#'     it.
 #'
 #' 4.  If you push this code to your CI server, it will already "work" because
 #'     all functions automatically skip tests when your `{MYPACKAGE}_KEY}`
```

Comment:
Not in this diff but there's one too many closing `}` in `` `{MYPACKAGE}_KEY}` ``